PULL_REQUEST_TEMPLATE.MD

```diff
@@ -9,5 +9,5 @@ This pull request adheres to the repository's [Code of Conduct](https://github.c
 - [ ] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
 - [ ] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
 - [ ] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
-- [ ] __Region__ details any restrictions to applicants based on geography
+- [ ] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md).
```

Comment:
Let's make this link to `/company-profiles/example.md#region` so that it goes directly to the right section.

Reply:
Agreed 👍

Reply:
This link still needs the `#region` anchor.

Reply:
Yes, you're totally right! Sorry!!! :grimacing:
Thank you for pointing this out!:thumbsup:

Reply:
This is the last change we need here, and then we can merge this.